myapp/Gemfile

```diff
@@ -58,6 +58,7 @@ group :development, :test do
   gem 'debug', platforms: %i[mri mingw x64_mingw]
   gem 'factory_bot_rails'
   gem 'rspec-rails'
+  gem 'pry-rails'
```

Comment:
Rails 7 uses debug gem as default, but I thought it's kinda uncomfortable to use.
So add `pry-rails` gem.